src/xcode/ENA/ENA/Source/Services/PPAccessControl/Model/SRSPreconditionError.swift

```diff
@@ -0,0 +1,59 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+enum SRSPreconditionError: Error {
+
+	/// Precondition: The device time check is correct
+	case deviceCheckError(PPACError)
+
+	/// Precondition: the app was installed less than 48h
+	case insufficientAppUsageTime
+	
+	/// Precondition: there was already a key submission without a registered test in the last 3 months
+	case positiveTestResultWasAlreadySubmittedWithin90Days
```

Comment:
Naming nitpick: as the 90 days are not hardcoded in the app and could be changed via app config, you could think about a name that doesn't include the 90 days

Reply:
good point